README.md

```diff
@@ -154,13 +153,12 @@ _**Important!** The [unpkg.com](https://unpkg.com), [jsDelivr](https://www.jsdel
 **Load the "attribution" build** _(using a module script)_
 
 ```html
-<!-- Append the `?module` param to load the module version of `web-vitals` -->
 <script type="module">
   import {
     onCLS,
     onINP,
     onLCP,
-  } from 'https://unpkg.com/web-vitals@4/dist/web-vitals.attribution.js?module';
```

Comment:
The use of `?module` here doesn't seem to have been useful anyway since `web-vitals.attribution.js` is already a module.

Reply:
And the use here or `?module` doesn't do anything, right?